components/nimbus/src/behavior.rs

```diff
@@ -14,13 +13,30 @@ use std::hash::{Hash, Hasher};
 
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub enum Interval {
+    Minutes,
     Hours,
     Days,
     Weeks,
     Months,
     Years,
 }
 
+impl Interval {
```

Comment:
This makes sense to move here, good call!

Reply:
Yep James suggested it on Wednesday so I figured I'd do it in this PR